src/org/redalert1741/steamworks/Robot.java

```diff
@@ -198,15 +224,15 @@ void StartLogging(String mode, DataLogger l)
 	void SetupLogging()
```

Comment:
It might be better to make the log file a parameter of SetupLogging so you can re-use it between teleop and autonomous.